addons/mail/static/src/new/thread/message.xml

```diff
@@ -2,10 +2,8 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.message" owl="1">
-    <div class="o-mail-message position-relative py-1 mt-1" t-att-class="{
+    <div class="o-mail-message position-relative py-1 mt-1 px-3" t-att-class="{
             'o-mail-is-author': message.isAuthor,
-            'px-3': !env.inChatWindow,
-            'px-1': env.inChatWindow,
```

Comment:
old code had these conditions, how comes they are no longer necessary?

Reply:
The template style do not 100% match before refactoring.
The `master-discuss-refactoring` branch had many differences already.
So we cannot just apply the same rules, as they made sense in old template but not in new template.

Here it just happens that `px-3` gives proper spacing in discuss app, but in chat window it's too close to chat window border. Maybe in old code we had some padding in content of chat window, but it doesn't matter to have exact same values. Style changes are easy to do, and it's good to have some freedom to change when we think this change is better.